templates/archive-faq.php

```diff
@@ -1,23 +1,58 @@
 <?php get_header(); ?>
 
 <article>
-	<div class="container my-5 ucf-faq-list">
-		<h1 class="ucf-faq-title mb-4">Frequently Asked Questions</h1>
+	<div class="<?php if( get_option( 'ucf_faq_include_athena_classes' ) ) echo "container my-5 "; ?>ucf-faq-list">
+		<h1 class="ucf-faq-title<?php if( get_option( 'ucf_faq_include_athena_classes' ) ) echo " mb-4"; ?>">Frequently Asked Questions</h1>
 		<?php
 		if ( have_posts() ) {
-			while ( have_posts() ) {
-				the_post();
+
+			$args = array(
```

Comment:
Avoid mixing `have_posts()` with custom `get_posts()` queries--only reference `have_posts()` if you plan on using the standard WordPress Loop to loop through and display queried posts.

If you have to use a custom `get_posts()` call here, check the count of the returned `$posts` array like we usually do before looping through it, instead of using `have_posts()`.

As an alternative, instead of re-fetching all FAQs here via `get_posts()`, I'd suggest accessing the list of posts already stored in the global `$wp_query` object when this archive is loaded to reduce the number of db queries.

e.g.

```php
global $wp_query;
if ( have_posts() ) {
	$posts = $wp_query->posts;
	$items = array();

	foreach ( $posts as $post ) {
		$topics = wp_get_post_terms( $post->ID, 'topic' );
			foreach( $topics as $topic ) {
				$items[$topic->name][] = $post;
			}
		}
	}

	...
}
```

However, doing this would probably require us to manually override how many results get returned in FAQ post type archives via `$wp_query` (so that `posts_per_page` is always `-1`, instead of whatever the default for post type archives is), since there's probably some limit enforced by default.  Doing this would be pretty simple to add though, and would probably also use the `pre_get_posts` hook, like we do here: https://github.com/UCF/UCF-FAQ-CPT/blob/rc-v1.0.0/templates/templates.php#L46-L53  (we may even just want to add the `posts_per_page` enforcement in that function).